### PR TITLE
Update WebUtilities.java

### PR DESCRIPTION
**Pull request by beyazgulsamil**
Add scrollAndClickElement

### CODE REVIEW

File: src/main/java/utils/WebUtilities.java [hunk 1]
```diff
@@ -400,7 +400,20 @@ public WebElement waitUntilElementIsVisible(WebElement element, long initialTime
...
         return element;
     }
-
+    /**
+     * Used to scroll the elements inside the container
+     *
+     * @param list
+     * @param buttonText
+     */
+    public void scrollInContainer(List<WebElement> list, String buttonText){
```
Comment (Umutayb) [inline src/main/java/utils/WebUtilities.java:409]: rename "buttonText" into "elementText"

File: src/main/java/utils/WebUtilities.java [hunk 2]
```diff
@@ -400,7 +400,20 @@ public WebElement waitUntilElementIsVisible(WebElement element, long initialTime
         }
         return element;
     }
-
+    /**
+     * Used to scroll the elements inside the container
+     *
+     * @param list
```
Comment (Umutayb) [inline src/main/java/utils/WebUtilities.java:406]: add param descriptions

**Review by Umutayb — CHANGES_REQUESTED** [review]

**Review by Umutayb — APPROVED** [review]
Great job, clean utility!

File: src/main/java/utils/WebUtilities.java (hunk 1 above)
Comment (beyazgulsamil) [reply src/main/java/utils/WebUtilities.java]: DONE

File: src/main/java/utils/WebUtilities.java (hunk 2 above)
Comment (beyazgulsamil) [reply src/main/java/utils/WebUtilities.java]: DONE

**Pull request merged**